src/Microsoft.Dnx.Compilation.CSharp/RoslynProjectReference.cs

```diff
@@ -151,7 +173,8 @@ public void EmitReferenceAssembly(Stream stream)
 
         public DiagnosticResult EmitAssembly(string outputPath)
         {
-            IList<ResourceDescription> resources = CompilationContext.Resources;
+            var resources = Enumerable.Empty<ResourceDescription>();
+            EmitResources(outputPath, out resources);
```

Comment:
Is there any reason that the result is ignored here? does that mean that errors don't show up when compiling the resource assembly? Can there be any errors?